internal/envoy/v3/route.go

```diff
@@ -122,6 +122,15 @@ func buildRoute(dagRoute *dag.Route, vhostName string, secure bool) *envoy_route
 		// redirect routes to *both* the insecure and secure vhosts.
 		route.Action = UpgradeHTTPS()
 	case dagRoute.DirectResponse != nil:
+		route.TypedPerFilterConfig = map[string]*anypb.Any{}
+
+		// Apply per-route authorization policy modifications.
+		if dagRoute.AuthDisabled {
+			route.TypedPerFilterConfig["envoy.filters.http.ext_authz"] = routeAuthzDisabled()
+		} else if len(dagRoute.AuthContext) > 0 {
+			route.TypedPerFilterConfig["envoy.filters.http.ext_authz"] = routeAuthzContext(dagRoute.AuthContext)
+		}
+
```

Comment:
I was debuting between disabling auth completely on a direct response or not. I left it for the users.

Reply:
@davinci26 do you mean you prefer a global control over this, or that the implementation here is sufficient?

Reply:
I think the current implmenetation makes sense and this is per route (even tho this is a direct response route) similar to how we have `AuthContext` per routes